deep-security-agent/recipes/default.rb

```diff
@@ -13,242 +13,191 @@
-		"fn" => "ds_agent.rpm",
-		32 => "https://#{dsm_agent_download_hostname}/software/agent/amzn1/i386/",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/amzn1/x86_64/",
-		},
-	"rhel_4" => {
-		"fn" => "ds_agent.rpm",
-		32 => "https://#{dsm_agent_download_hostname}/software/agent/RedHat_2.6.9_22.EL_i686/i386/",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/RedHat_2.6.9_34.EL_x86_64/x86_64/",
-		},
-	"rhel_5" => {
-		"fn" => "ds_agent.rpm",
-		32 => "https://#{dsm_agent_download_hostname}/software/agent/RedHat_EL5/i386/",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/RedHat_EL5/x86_64/",
-		},
-	"rhel_6" => {
-		"fn" => "ds_agent.rpm",
-		32 => "https://#{dsm_agent_download_hostname}/software/agent/RedHat_EL6/i386/",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/RedHat_EL6/x86_64/",
-		},
-	"rhel_7" => {
-		"fn" => "ds_agent.rpm",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/RedHat_EL7/x86_64/",
-		},		
-	"suse_10" => {
-		"fn" => "ds_agent.rpm",
-		32 => "https://#{dsm_agent_download_hostname}/software/agent/SuSE_10/i386/",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/SuSE_10/x86_64/",
-		},
-	"suse_11" => {
-		"fn" => "ds_agent.rpm",
-		32 => "https://#{dsm_agent_download_hostname}/software/agent/SuSE_11/i386/",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/SuSE_11/x86_64/",
-		},
-	"suse_12" => {
-		"fn" => "ds_agent.rpm",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/SuSE_12/x86_64/",
-		},				
-	"win" => {
-		"fn" => "ds_agent.msi",
-		32 => "https://#{dsm_agent_download_hostname}/software/agent/Windows/i386/",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/Windows/x86_64/",
-		},
-	"ubuntu_10" => {
-		"fn" => "ds_agent.deb",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/Ubuntu_10.04/x86_64/",
-		},
-	"ubuntu_12" => {
-		"fn" => "ds_agent.deb",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/Ubuntu_12.04/x86_64/",
-		},
-	"ubuntu_14" => {
-		"fn" => "ds_agent.deb",
-		64 => "https://#{dsm_agent_download_hostname}/software/agent/Ubuntu_14.04/x86_64/",
-		},	
-	}
+dsm_agent_download_hostname 	= node['deep_security_agent']['dsm_agent_download_hostname']
+dsm_agent_download_port 		= node['deep_security_agent']['dsm_agent_download_port']
+ignore_ssl_validation 			= node['deep_security_agent']['ignore_ssl_validation']
+dsm_agent_activation_hostname 	= node['deep_security_agent']['dsm_agent_activation_hostname']
+dsm_agent_activation_port 		= node['deep_security_agent']['dsm_agent_activation_port']
+tenant_id 						= node['deep_security_agent']['tenant_id']
+token 				            = node['deep_security_agent']['token']
+policy_id 						= node['deep_security_agent']['policy_id']
+
+PACKAGE_RPM = 'ds_agent.rpm'.freeze
+PACKAGE_DEB = 'ds_agent.deb'.freeze
+PACKAGE_MSI = 'ds_agent.msi'.freeze
 
 # Determine the correct agent based on information reported by Ohai
```

Comment:
Should 'Ohai' be mentioned?

Reply:
Can be removed, but for code readability it is better to retain it.